### ISSUE TITLE
Text Editor Issue

### ISSUE THREAD

**Issue by Tangol** [villagedefrance/OpenCart-Overclocked#144]
Please note that the recent commits (probably the ones related to the HTML Module) introduced a bug at **CKEditor** and the simple text areas.
Whenever I try to insert text, I get the following error.

`Article Content must be greater than 3 characters !`
![err](https://user-images.githubusercontent.com/3336728/36815937-042c0c96-1ce5-11e8-95b3-47a08cb2688e.jpg)

The same for the **Reviews**:
![err2](https://user-images.githubusercontent.com/3336728/36816226-e1276884-1ce5-11e8-8457-2f1e673847e2.jpg)

Also, I can't update any product descriptions, as it doesn't save the modifications.
So basically all text areas on the website don't work anymore.

Tested on https://github.com/villagedefrance/OpenCart-Overclocked/commit/d55353bcf6bf2877afbc04c0a5aca06e05ab614e

**Comment by Tangol** [comment]
Issue partially fixed.

Only the **Reviews** problem remains:
![err2](https://user-images.githubusercontent.com/3336728/36816226-e1276884-1ce5-11e8-8457-2f1e673847e2.jpg)

**Comment by Tangol** [comment]
Issue fixed.

No bug at **Reviews** - didn't realize I only had 24 characters in the review. 😁 

Can be closed.
Thanks!
